llama_stack/providers/tests/safety/fixtures.py

```diff
@@ -31,23 +28,19 @@ def safety_model(request):
 
 
 @pytest.fixture(scope="session")
-def safety_meta_reference(safety_model) -> ProviderFixture:
+def safety_llama_guard(safety_model) -> ProviderFixture:
     return ProviderFixture(
         providers=[
             Provider(
-                provider_id="meta-reference",
-                provider_type="meta-reference",
-                config=SafetyConfig(
-                    llama_guard_shield=LlamaGuardShieldConfig(
-                        model=safety_model,
-                    ),
-                ).model_dump(),
+                provider_id="inline::llama-guard",
+                provider_type="inline::llama-guard",
+                config=LlamaGuardConfig(model=safety_model).model_dump(),
             )
         ],
     )
 
 
-SAFETY_FIXTURES = ["meta_reference", "remote"]
+SAFETY_FIXTURES = ["llama_guard", "remote"]
```

Comment:
Shall we also add `prompt_guard` as a fixture?

Reply:
good idea